src/main/resources/layouts/manage-project.fxml

```diff
@@ -25,23 +25,25 @@
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
       <RowConstraints vgrow="SOMETIMES" />
+      <RowConstraints vgrow="SOMETIMES" />
   </rowConstraints>
    <children>
       <Label text="Name:" />
-      <Label text="Color:" GridPane.rowIndex="3" />
-      <Label text="IsWork:" GridPane.rowIndex="4" />
-      <Label text="SortIndex:" GridPane.rowIndex="5" />
+      <Label text="Color:" GridPane.rowIndex="4" />
+      <Label text="IsWork:" GridPane.rowIndex="5" />
+      <Label text="SortIndex:" GridPane.rowIndex="6" />
       <TextField fx:id="nameTextField" GridPane.columnIndex="1" />
-      <ColorPicker fx:id="textFillColorPicker" GridPane.columnIndex="1" GridPane.rowIndex="3" />
-      <CheckBox fx:id="isWorkCheckBox" mnemonicParsing="false" selected="true" GridPane.columnIndex="1" GridPane.rowIndex="4" />
-      <Spinner fx:id="sortIndexSpinner" GridPane.columnIndex="1" GridPane.rowIndex="5" />
-      <TextArea fx:id="descriptionTextArea" prefHeight="120.0" prefWidth="300.0" GridPane.columnIndex="1" GridPane.rowIndex="1" />
-      <VBox alignment="CENTER_LEFT" GridPane.rowIndex="1">
+      <ColorPicker fx:id="textFillColorPicker" GridPane.columnIndex="1" GridPane.rowIndex="4" />
+      <CheckBox fx:id="isWorkCheckBox" mnemonicParsing="false" selected="true" GridPane.columnIndex="1" GridPane.rowIndex="5" />
+      <Spinner fx:id="sortIndexSpinner" GridPane.columnIndex="1" GridPane.rowIndex="6" />
+      <TextArea fx:id="descriptionTextArea" prefHeight="120.0" prefWidth="300.0" GridPane.columnIndex="1" GridPane.rowIndex="2" />
+      <VBox alignment="CENTER_LEFT" GridPane.rowIndex="2">
          <children>
             <Label text="Description:" />
             <Label disable="true" prefWidth="100.0" text="(will be matched at search)" wrapText="true" />
          </children>
       </VBox>
+      <Label fx:id="validateTextAlert" prefHeight="17.0" prefWidth="201.0" text="Please enter a project name!" GridPane.columnIndex="1" GridPane.rowIndex="1" />
```

Comment:
let's remove the "!" from the hint :) Sounds more friendly then - especially as the hint appears directly when opening the dialog

Reply:
Removed it :)